lib/Elevate/Components/Imunify.pm

```diff
@@ -39,6 +41,40 @@ sub post_leapp ($self) {
 
     $self->run_once('_restore_imunify_features');
 
+    # this is happening after Imunify360 component
+    $self->run_once("_restore_imunify_packages");
+
+    return;
+}
+
+sub _capture_imunify_packages ($self) {
+    my $installed    = cpev::yum_list();
+    my @packages_imu = grep { m/^imunify-/ } keys %$installed;
+
+    my @packages_from_repo = cpev::get_installed_rpms_in_repo('imunify');
+    my %all = map { $_ => 1 } @packages_imu, @packages_from_repo;
+
+    my @packages = sort keys %all;
+
+    return unless scalar @packages;
+
+    cpev::update_stage_file( { 'reinstall' => { 'imunify_packages' => \@packages } } );
+
+    return;
+}
+
+sub _restore_imunify_packages ($self) {
+
+    # try to reinstall missing Imunify packages which were previously installed
+
+    return unless my $packages = cpev::read_stage_file('reinstall')->{'imunify_packages'};
+
+    foreach my $pkg (@$packages) {
+        next unless Cpanel::Pkgr::is_installed($pkg);
+        INFO("Try to reinstall Imunify package: $pkg");
+        $self->ssystem( qw{ /usr/bin/dnf -y install }, $pkg );
```

Comment:
some of the packages exist on 7 but shouldn't be installed on 8

Reply:
it s only the imunify packages:
```
{
   "reinstall" : {
      "imunify_packages" : [
         "imunify-antivirus",
         "imunify-antivirus-cpanel",
         "imunify-common",
         "imunify-notifier"
      ]
   }
}
```

Reply:
For example not restoring the `imunify-antivirus-cpanel` lead to a broken WHM UI when trying to access to the plugin